### DISCUSSION TITLE
Able to organize collections and favourites by drag and drop

### DISCUSSION THREAD

**Discussion by bigint** [toeverything/AFFiNE#6211] [General]
### Description

Would be super cool if able to organize collections and favourites by drag and drop

<img width="337" alt="image" src="https://github.com/toeverything/AFFiNE/assets/69431456/e30fe1f3-3d36-4496-9f94-700bfc34c17a">

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by CatsJuice** [comment]
Do you mean drag to reorder items in Favourites/collections? I think we already have relevant plans.

BTW, I've moved this issue to discussion since it's more like a feature request.
